Resources/Locale/en-US/round-end/round-end-system.ftl

```diff
@@ -1,6 +1,10 @@
 ## RoundEndSystem
 
-round-end-system-shuttle-called-announcement = An emergency shuttle has been sent. ETA: {$time} {$units}.
+round-end-system-shuttle-called-announcement =  An emergency shuttle has been sent. ETA: {$time} {$units}.
+round-end-system-shuttle-called-announcement-reason = 
+    An emergency shuttle has been sent. ETA: {$time} {$units}.
+
+    Nature of emergency: {$reason}
```

Comment:
you can probably just combine these by doing a conditional check or something instead of duping the string. or even just doing a formatted message and appending a second string if the reason field is not null